src/rules/prefer-importing-jest-globals.ts

```diff
@@ -0,0 +1,189 @@
+
+  return isModule
+    ? `import { ${allImportsFormatted} } from '@jest/globals';`
+    : `const { ${allImportsFormatted} } = require('@jest/globals');`;
+};
+
+export default createRule({
+  name: __filename,
+  meta: {
+    docs: {
+      category: 'Best Practices',
+      description: 'Prefer importing Jest globals',
+      recommended: false,
+    },
+    messages: {
+      preferImportingJestGlobal: `Import the following Jest functions from '@jest/globals': {{ jestFunctions }}`,
+    },
+    fixable: 'code',
+    type: 'problem',
+    schema: [],
+  },
+  defaultOptions: [],
+  create(context) {
+    const importedJestFunctions: string[] = [];
+    const usedJestFunctions: ParsedJestFnCall[] = [];
+
+    return {
+      CallExpression(node) {
+        const jestFnCall = parseJestFnCall(node, context);
+
+        if (!jestFnCall) {
+          return;
+        }
+
+        if (jestFnCall.head.type === 'import') {
+          importedJestFunctions.push(jestFnCall.name);
+        }
+
+        usedJestFunctions.push(jestFnCall);
+      },
+      'Program:exit'() {
+        const jestFunctionsToReport = usedJestFunctions.filter(
+          jestFunction => !importedJestFunctions.includes(jestFunction.name),
+        );
+
+        if (!jestFunctionsToReport.length) {
+          return;
+        }
+        const jestFunctionsToImport = jestFunctionsToReport.map(
+          jestFunction => jestFunction.name,
+        );
+        const reportingNode = jestFunctionsToReport[0].head.node;
+
+        const jestFunctionsToImportFormatted = jestFunctionsToImport.join(', ');
+
+        const isModule = context.parserOptions.sourceType === 'module';
+
+        context.report({
+          node: reportingNode,
+          messageId: 'preferImportingJestGlobal',
+          data: { jestFunctions: jestFunctionsToImportFormatted },
+          fix(fixer) {
+            const sourceCode = getSourceCode(context);
+            const [firstNode] = sourceCode.ast.body;
+
+            // check if "use strict" directive exists
+            if (
+              firstNode.type === AST_NODE_TYPES.ExpressionStatement &&
+              firstNode.expression.type === AST_NODE_TYPES.Literal &&
+              firstNode.expression.value === 'use strict'
+            ) {
+              return fixer.insertTextAfter(
+                firstNode,
+                `\n${createFixerImports(isModule, jestFunctionsToImport)}`,
+              );
+            }
+
+            const importNode = sourceCode.ast.body.find(
+              node =>
+                node.type === AST_NODE_TYPES.ImportDeclaration &&
+                node.source.value === '@jest/globals',
+            );
+
+            if (
+              importNode &&
+              importNode.type === AST_NODE_TYPES.ImportDeclaration
+            ) {
+              const existingImports = importNode.specifiers.reduce<string[]>(
+                (imports, specifier) => {
+                  /* istanbul ignore else */
```

Comment:
this can be covered by e.g. a default import:

```
import describe from '@jest/globals';
describe("suite", () => {
  test("foo");
  expect(true).toBeDefined();
})
```

General tip for helping track down coverage: look at what types something can be.

In this case, `specifier` is of type `ImportClause` which is:

```
export declare type ImportClause = ImportDefaultSpecifier | ImportNamespaceSpecifier | ImportSpecifier;
```

So you want to play around with those three types of things to get coverage - and it seems in this case just having a default import `@jest/globals` is enough